nss-day-cohort-39/holiday-road-the-dudes
# Delete button and State Change work

### Changes
- added delete button to saved itineraries on itinerary.js
- added state change to itinerarydataProvider.js and dispatched to event hub
- listen for state change on itineraryList and re render saved Itineraries

### testing
- save itinerary and confirm it immediately shows up under saved trips without have to reload the page
- delete itinerary and confirm it immediately disappears from saved trips without reload
